demo/index.html

```diff
@@ -25,16 +25,17 @@
   <link rel="import" href="../../paper-dialog-scrollable/paper-dialog-scrollable.html">
   <link rel="import" href="../../paper-styles/color.html">
   <link rel="import" href="../../paper-styles/demo-pages.html">
```

Comment:
I think this import can be removed; the `custom-style` with `include="demo-pages-shared-styles"` covers it, right?